code/processes/rdb.q

```diff
@@ -121,7 +121,7 @@ endofday:{[date;processdata]
 	hdbs:distinct raze {exec w from .servers.getservers[x;y;()!();1b;0b]}'[`proctype`procname;(hdbtypes;hdbnames)];
 	$[.finspace.enabled;
                      .finspace.notifyhdb[;changeset] each .finspace.hdbclusters;
-                     notifyhdb[;date] each hdbs;
+	             notifyhdb[;date] each hdbs
```

Comment:
indentation is still partially changed here (now has one tab at start of line, previously was all spaces), if you can change that back please

Reply:
I've changed it again now. I just changed it entirely to spaces in Fixing indentation #3. Hopefully thats it now